### DISCUSSION TITLE
用kodi播放时时不时会停止播放

### DISCUSSION THREAD

**Discussion by liyiranlab** [messense/aliyundrive-webdav#631] [Q&A]
### 问题描述

用kodi播放时时不时会停止播放，暂停播放后再播放必停止播放。

### 重现步骤

1.用kodi正常播放，时不时会停止播放；
2.用kodi正常播放，按暂停，恢复播放后一般都会停止播放。

### 版本

1.010.1

### 运行平台

OpenWrt

### refresh token 获取方式

网页版二维码

### 日志

2022-10-27T07:32:46.363910Z  WARN reqwest_retry::middleware: Retry attempt #0. Sleeping 36.220311ms before the next attempt

**Comment by liyiranlab** [comment]
问题是多方面的，即使将网络优化到最好，用302模式依然会自然断开，排除有的时候是视频有问题。

**Reply by liyiranlab** [reply]
现在已经是2.3.3依然有这个情况，我再尝试不开启302模式，正常情况下看看是否还会断开。

**Reply by oooopera** [reply]
> 现在已经是2.3.3依然有这个情况，我再尝试不开启302模式，正常情况下看看是否还会断开。

解决了吗，遇到了相同的问题
